_data/other.yml

```diff
@@ -538,6 +538,10 @@ websites:
     tfa:
       - totp
     doc: https://support.usabilla.com/hc/en-us/articles/115003081611
+    
+  - name: Vivaldi
+    url: https://vivaldi.com/
+    img: vivaldi.png
```

Comment:
```suggestion
    img: vivaldi.png
    facebook: vivaldi.browser
    twitter: vivaldibrowser
```

Reply:
Thank you. I had overlooked adding social media links. Are you requesting a change to vivaldi.png? I had downloaded the favicon, cropped it, and resized it. Is that a violation of anything?